python/rikai/types/vision.py

```diff
@@ -104,6 +105,20 @@ def from_pil(img: PILImage, uri: Union[str, Path]) -> Image:
                 copy(fobj.name, uri)
         return Image(uri)
 
+    @staticmethod
+    def from_bytes(img: bytes, uri: Union[str, Path]) -> Image:
+        """Create an image in memory from a byte string.
+
+        Parameters
+        ----------
+        img : bytes
+            An PIL Image instance
```

Comment:
Would like to confirm that the input are bytes, not a PIL instance?

Reply:
thanks, updated.